planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/structs/debug.hpp

```diff
@@ -25,10 +25,20 @@
 
 #include <limits>
 #include <string>
+#include <vector>
 
 namespace autoware::behavior_path_planner::lane_change
 {
 using utils::path_safety_checker::CollisionCheckDebugMap;
+
+struct LaneChangeMetricsDebug
```

Comment:
It's already in the lane change namespace, so do we still need the "LaneChange" in the name?

Reply:
hmm true maybe `PhaseMetricsDebug` ?